networkit/graph.pyx

```diff
@@ -901,6 +908,47 @@ cdef class Graph:
 			raise Exception("Attribute name has to be a string")
 		self._this.detachNodeAttribute(stdstring(name))
 
+	def from_coordinates(self, np.ndarray[DINT_t, ndim=1] row, np.ndarray[DINT_t, ndim=1] col, np.ndarray[DDOUBLE_t, ndim=1] data):
+		"""
+		from_coo_matrix(input)
```

Comment:
Typo? 

Also I think the name `from_coordinates` can be improved, cause it might be misleading. How about the originally suggested `from_coo_arrays`?